test/servico/Cst21Test.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package servico;
+
+import auxiliar.Perfil;
+import datamapper.PopulateDB;
+import dominio.Usuario;
+import modelo.UsuarioModel;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+/**
+ *
+ * @author Victor Messina
+ */
+public class Cst21Test {
+
+
+    @BeforeClass
+    public static void setUpClass() throws Exception {
+        PopulateDB.recreateDB("prosub", "root", "");
+        PopulateDB.populateUseCaseTest();
+    }
+
+    @Before
+    public void setUp() {
+
+        AdministracaoDeUsuariosService aus;
+
+        aus = new AdministracaoDeUsuariosService();
+
+        Usuario usuario1 = new Usuario();
+
+        usuario1.setUsuario("ProfessorTest");
+        usuario1.setSenha("123456");

```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [The String "123456" appears 3 times in the file.](https://www.codacy.com/app/calebepb/substituicao/file/2777777468/issues/source?bid=3364462&fileBranchId=3364491#l40)